samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -0,0 +1,161 @@
+                                    myWords.items[i].font.highlightColor = "#FFFF00";
+                                    return createTimerPromise(200).then(context.sync);
+                                })
+                            })
+                    })
+            })
+                .catch(OfficeHelpers.Utilities.log);
+
+            function createTimerPromise(ms) {
+                return new OfficeExtension.Promise(function (resolve) {
+                    setTimeout(resolve, ms);
+                })
+            }
+
+            function forEach(collection, handler) {
+                var promise = new OfficeExtension.Promise(function (resolve) { resolve(); });
+                collection.items.forEach(function (item, index) {
+                    promise = promise.then(function () {
+                        return handler(item, index);
+                    })
+                });
+                return promise;
+            }
+        }
+
+        function setup() {
+            Word.run(function (context) {
+                // lets insert a couple of paragraphs to illustrate the point..
+                context.document.body.clear();
+                context.document.body.insertParagraph("Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.", "start");
+                context.document.body.paragraphs.getLast().insertText("To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching Online cover page, header, and sidebar. Click Insert and then choose the Online elements you want from the different Online galleries.", "replace");
+                return context.sync();
+            })
+        }
+    language: typescript
+template:
+    content: |4
+          <h2 class="ms-font-m">This sample demonstrates splitting and traversing ranges.</h2>
+
+
+            <div id="setup-container">
+                <p class="ms-font-m">Click "setup" to insert sample text and between runs to reset the sample.</p>
+
+                <button id="setup" class="ms-Button">
+                    <span class="ms-Button-label">Setup</span>
+                </button>
+            </div>
+
+
+            <div id="samples-container">
+                <p class="ms-font-m">Sample snippets to try:</p>
+
+                <button id="highlight" class="ms-Button">
+                    <span class="ms-Button-label">Highlight word by word</span>
+                </button>   
+
+                
+            </div>
+    language: html
+style:
+    content: |4
+            body {
+                margin: 0;
+                padding: 10px;
+            }
+
+
+            /* Button customization, including overwriting some Fabric defaults */
```

Comment:
i see this as a p3 change. i will prioritize adding more samples and will fix it later (it requires changing all the samples) also it should be ok to use CSS.